src/cache.rs

```diff
@@ -831,18 +831,37 @@ impl Cache {
         self.terms.get(&file_id).map(|CachedTerm { term, .. }| term)
     }
 
+    /// Retrieve the FileId for a given standard libray module.
+    pub fn get_submodule_file_id(&self, module: StdlibModule) -> Option<FileId> {
```

Comment:
Suggestion, instead of `submodule` (which doesn't really have a definition currently):

```suggestion
    pub fn get_stdlib_module_file_id(&self, module: StdlibModule) -> Option<FileId> {
```